### PR TITLE
doc: Add documentation about cm/shoot-info

### PR DESCRIPTION
**Pull request by poelzi**
Add documentation about the configmap gardener creates on the shoot
cluster with cluster information.

**How to categorize this PR?**
/area documentation
/kind enhancement

**What this PR does / why we need it**:

Feature currently not documented and very hard to find.

**Release note**:
```doc user
Documentation about the Shoot info ConfigMap is now added.
```

### CODE REVIEW

**Comment by gardener-robot-ci-1** [comment]
Thank you @poelzi for your contribution. Before I can start building your PR, a member of the organization must set the required label(s) {'reviewed/ok-to-test'}. Once started, you can check the build status in the PR checks section below.

File: docs/usage/shoot_info.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Shoot Info
```
Comment (rfranzke) [inline docs/usage/shoot_info.md]: Please add the document to the index (https://github.com/gardener/gardener/tree/master/docs#usage)

**Review by rfranzke — DISMISSED** [review]
/lgtm
/reviewed ok-to-test

File: docs/usage/shoot_info.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+| `maintenanceEnd`     | Time when maintenance window ends                                                                                  |
+| `nodeNetwork`        | CIDR of cluster node network                                                                                       |
+| `podNetwork`         | CIDR of pod network                                                                                                |
+| `projectName`        | Project name under which this cluster resides.                                                                     |
+| `provider`           | Provider extension used for creating this cluster                                                                  |
+| `region`             | Region this cluster is placed                                                                                      |
+| `serviceNetwork`     | CIDR of service network                                                                                            |
+| `shootName`          | Name of the shoot cluster                                                                                          |
```
Comment (ialidzhikov) [inline docs/usage/shoot_info.md]:
```diff
diff --git a/docs/README.md b/docs/README.md
index ce65e5d83..74e21d885 100644
--- a/docs/README.md
+++ b/docs/README.md
@@ -50,7 +50,7 @@
 * [Shoot Networking](usage/shoot_networking.md)
 * [Shoot Maintenance](usage/shoot_maintenance.md)
 * [Shoot Status](usage/shoot_status.md)
-* [Shoot Info](usage/shoot_info.md)
+* [Shoot Info `ConfigMap`](usage/shoot_info_configmap.md)
 * [Shoot Updates and Upgrades](usage/shoot_updates.md)
 * [Supported Kubernetes versions](usage/supported_k8s_versions.md)
 * [Tolerations](usage/tolerations.md)
diff --git a/docs/usage/shoot_info.md b/docs/usage/shoot_info.md
deleted file mode 100644
index 272c4b7fd..000000000
--- a/docs/usage/shoot_info.md
+++ /dev/null
@@ -1,26 +0,0 @@
-# Shoot Info
-
-Inside a shoots kubernetes cluster, gardener provides a [ConfigMap](https://kubernetes.io/docs/concepts/configuration/configmap/)
-with information about the cluster itself.
-The map is in the namespace `kube-system` named `shoot-info`.
-
-Example: `kubectl get configmap -n kube-system shoot-info -o yaml`
-
-## Fields
-
-Following fields are provided:
-
-| Field name           | Description                                                                                                        |
-| -------------------- | ------------------------------------------------------------------------------------------------------------------ |
-| `domain`             | Domain under which this cluster is reachable.                                                                      |
-| `extensions`         | List of activated extensions.                                                                                      |
-| `kubernetesVersion`  | Current kubernetes version deployed                                                                                |
-| `maintenanceBegin`   | Time when maintenance window starts                                                                                |
-| `maintenanceEnd`     | Time when maintenance window ends                                                                                  |
-| `nodeNetwork`        | CIDR of cluster node network                                                                                       |
-| `podNetwork`         | CIDR of pod network                                                                                                |
-| `projectName`        | Project name under which this cluster resides.                                                                     |
-| `provider`           | Provider extension used for creating this cluster                                                                  |
-| `region`             | Region this cluster is placed                                                                                      |
-| `serviceNetwork`     | CIDR of service network                                                                                            |
-| `shootName`          | Name of the shoot cluster                                                                                          |
diff --git a/docs/usage/shoot_info_configmap.md b/docs/usage/shoot_info_configmap.md
new file mode 100644
index 000000000..2825bc66d
--- /dev/null
+++ b/docs/usage/shoot_info_configmap.md
@@ -0,0 +1,30 @@
+# Shoot Info `ConfigMap`
+
+## Overview
+
+Gardenlet maintains a [ConfigMap](https://kubernetes.io/docs/concepts/configuration/configmap/) inside the Shoot cluster that contains information about the cluster itself. The ConfigMap is named `shoot-info` and located in the `kube-system` namespace.
+
+## Fields
+
+The following fields are provided:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: shoot-info
+  namespace: kube-system
+data:
+  domain: crazy-botany.core.my-custom-domain.com     # .spec.dns.domain field from the Shoot resource
+  extensions: foobar,foobaz                          # List of extensions that are enabled
+  kubernetesVersion: 1.15.1                          # .spec.kubernetes.version field from the Shoot resource
+  maintenanceBegin: 220000+0100                      # .spec.maintenance.timeWindow.begin field from the Shoot resource
+  maintenanceEnd: 230000+0100                        # .spec.maintenance.timeWindow.end field from the Shoot resource
+  nodeNetwork: 10.250.0.0/16                         # .spec.networking.nodes field from the Shoot resource
+  podNetwork: 100.96.0.0/11                          # .spec.networking.pods field from the Shoot resource
+  projectName: dev                                   # .metadata.name of the Project                               
+  provider: <some-provider-name>                     # .spec.provider.type field from the Shoot resource
+  region: europe-central-1                           # .spec.region field from the Shoot resource
+  serviceNetwork: 100.64.0.0/13                      # .spec.networking.services field from the Shoot resource
+  shootName: crazy-botany                            # .metadata.name from the Shoot resource
+```

```

**Review by ialidzhikov — COMMENTED** [review]
Thanks for the PR!

I added a diff as inline comment that contains my styling suggestions.

File: docs/usage/shoot_info.md (hunk 2 above)
Comment (poelzi) [reply docs/usage/shoot_info.md]: done

**Review by rfranzke — APPROVED** [review]

**Review by ialidzhikov — APPROVED** [review]
/lgtm

**Pull request merged**